canvas_course_info/requirements/aws.txt

```diff
@@ -1,9 +1,9 @@
-Django==1.8.15
-dj-static==0.0.6
-django-redis-cache==1.7.0
+Django==1.10.5
+django-redis-cache==1.7.1
 git+https://github.com/harvard-dce/dce_lti_py.git@v0.7.4#egg=dce-lti-py==0.7.4
 git+https://github.com/harvard-dce/django-auth-lti.git@dce-lti-py#egg=django-auth-lti==1.0
+git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-ui.git@v1.3.1#egg=django-icommons-ui==1.3.1
 hiredis==0.2.0
 redis==2.10.5
-requests==2.11.1
-voluptuous==0.8.7
+requests==2.13.0
+voluptuous==0.9.3
```

Comment:
Nothing with respect to the `voluptuous` package changed in this PR, correct. Just trying to keep as many libraries as possible up-to-date. Unit tests do some basic exercising of the module that `voluptuous` stars in, so hopefully that's enough to ensure the update didn't break anything. :)

When I do fixes/updates, I typically try to go through all dev and prod libraries and look at the version diffs, determine the risk or benefit to upgrade, and bump where I feel it's not too risky. It's always a mixed bag -- for instance, we can't upgrade django-debug-toolbar at the moment without assessing the best way to configure it (see [TLT-2980](https://jira.huit.harvard.edu/browse/TLT-2980) -- would love your opinion on that!); we also haven't assessed Selenium 3 (see [TLT-2905](https://jira.huit.harvard.edu/browse/TLT-2905), so not upgrading that, etc etc.

Further thoughts welcome, including as always downsides to my approach.